src/qml/qmlconfigproxy.h

```diff
@@ -24,6 +24,10 @@ class QmlConfigProxy : public QObject {
     Q_INVOKABLE QVariantList getTrackColorPalette();
     Q_INVOKABLE int getMultiSamplingLevel();
     Q_INVOKABLE bool useAcceleration();
+    Q_INVOKABLE QVariant getValue(const QString& group,
+            const QString& item,
+            const QVariant& default_value) const;
+    Q_INVOKABLE void setValue(const QString& group, const QString& item, const QVariant& value);
```

Comment:
I'm unsure if we want to expose this directly. I'm at least iffy about `setValue`. Ideally the configproxy would remain strongly typed...